orc8r/cloud/go/services/orchestrator/obsidian/models/swagger.v1.yml

```diff
@@ -1410,6 +1412,17 @@ definitions:
         maximum: 1
         example: 0.5
 
+  network_state_config:
+    type: object
+    description: Network state configuration
+    properties:
+      sync_interval:
+        description: sync interval in seconds
+        type: number
+        format: uint32
+        minimum: 0
```

Comment:
Actual minimum should be 1m -- anything less than that could cause agw to get into weird states